packages/discord.js/src/structures/Channel.js

```diff
@@ -27,7 +27,8 @@ class Channel extends Base {
      * The type of the channel
      * @type {ChannelType}
      */
-    this.type = type ?? 'UNKNOWN';
+    // TODO: Better way of handling unknown?
```

Comment:
We won't have Channels without types anyways, let alone unknown types.

Reply:
If a new channel type is introduced it would be unknown wouldn’t it…